contents/docs/self-host/index.mdx

```diff
@@ -7,26 +7,32 @@ showTitle: true
 
 import Disclaimer from './deploy/snippets/disclaimer'
 
-Getting a production environment of PostHog up and running is probably one of the first things you want to do! This section goes through the different options available depending on where you'll be hosting PostHog, and includes guides on several of the most popular services.
+The fastest and most reliable way to get started with PostHog is to use [PostHog Cloud](/docs/getting-started/cloud).
+
+However, if you're a large company with the need to self-host or an engineer looking to use the open-source version in funky non-production way then you're in the right place!
+
+## PostHog Open-Source
 
 <Disclaimer />
 
-## Deployment options
+The core of PostHog is open-source and always will be. You can self-host PostHog the open-source version in weird and wonderful ways such as running it in your basement to [monitor a home IoT system](https://gist.github.com/mariusandra/d8add5fc7f32bedeae45d9ce8eb75b35).
```

Comment:
That link is to an article that goes out of its way to set up a kubernetes cluster (in the basement)... so literally the opposite of what this article is suggesting.